addon/remote/paged-remote-array.js

```diff
@@ -126,6 +126,12 @@ export default Ember.ArrayProxy.extend(PageMixin, Ember.Evented, ArrayProxyPromi
     }
   }),
 
+  reload: function() {
+    var promise = this.fetchContent();
+    this.set("promise", promise);
```

Comment:
Where is the reload? Aren't you missing a call to https://www.emberjs.com/api/ember-data/2.12/classes/DS.AdapterPopulatedRecordArray/methods/update?anchor=update ?

Reply:
From @synaptiko
```
to explain a bit why we went with `fetchContent` instead of `update`, we realized that additional logic in `fetchContent` makes sense also for reload because it's handling `loading` state and it will also update the `meta` which is something we want when adding/deleting items, there can be new page or page can disappear
```

I'm approving